service/host_test.go

```diff
@@ -195,7 +195,7 @@ func TestHostShutdown_RunningService(t *testing.T) {
 func TestHostShutdown_CloseSuccessful(t *testing.T) {
 	sh := makeRunningHost()
 	sh.serviceCore.metricsCore = metricsCore{
-		scope:            tally.NoopScope,
+		metrics:          tally.NoopScope,
```

Comment:
I see what you mean. The reason I finally went for this is if you look at service, we'll have
if svc.Metrics() == nil {
  svc.scope = tally.Scope
}

Checking Metrics() and setting scope appears unrelated to someone reading it. That's why I decided to keep them uniform.